packages/blockchain-wallet-v4-frontend/src/scenes/Login/template.js

```diff
@@ -120,7 +120,11 @@ const Login = (props) => {
         </FormGroup>
         <FormGroup>
           <LoginButton type='submit' nature='primary' fullwidth uppercase disabled={submitting || invalid}>
-            <FormattedMessage id='scenes.login.submit' defaultMessage='Log in' />
+            {
+              busy
+                ? <HeartbeatLoader height='20px' width='20px' color='white' />
+                : <FormattedMessage id='scenes.login.submit' defaultMessage='Log in' />
```

Comment:
@schnogz just FYI looks like this was done, know you were thinking of working on this.